test_strategy.md

```diff
@@ -0,0 +1,57 @@
+# Test strategy
+
+## Design test
+These are the checks and tests expected to be provided and maintained by the
```

Comment:
perhaps "expected to be provided" is a bit strong at this point.

Reply:
how about "the tests below have proven to be useful and are recommended for developing operators"

Reply:
I cannot feel confident to propose non trivial patches to operators not having this test coverage. I do believe not having this test coverage also slow other developers down providing those patches.

Reply:
I think that what @dprince wrote makes sense for me. We shouldn't be too strong with wording and let teams to develop testing strategy in own way/time/etc.
I understand this document as something what is/should be final goal for all teams but it should give teams more freedom when they will implement that.